### PR TITLE
Swift handling in CUBE and helper scripts

### PR DESCRIPTION
**Pull request by rudolphpienaar**
This PR adds swift support in CUBE itself as well as minor changes to JSON call to pfcon. Two helper scripts that can upload data to swift and list/pull data are also part of this PR.

### CODE REVIEW

File: chris_backend/plugins/services/manager.py [hunk 1]
```diff
@@ -266,6 +267,7 @@ def run_plugin_app(self, plugin_inst, parameter_dict, **kwargs):
             inputdir    = inputdirManagerFS,
             outputdir   = outputdirManagerFS,
             IOPhost     = self.str_IOPhost,
+            quiet       = settings.CHRIS_DEBUG['quiet']
```
Comment (NicolasRannou) [inline chris_backend/plugins/services/manager.py]: @rudolphpienaar missing quote

File: chris_backend/plugins/services/manager.py [hunk 2]
```diff
@@ -266,6 +267,7 @@ def run_plugin_app(self, plugin_inst, parameter_dict, **kwargs):
...
             gpuLimit    = int_gpuLimit,
             quiet       = True
```
Comment (NicolasRannou) [inline chris_backend/plugins/services/manager.py]: "quiet" already defined L270

**Comment by danmcp** [comment]
@rudolphpienaar Please squash the commits.

File: chris_backend/plugins/services/manager.py (hunk 1 above)
Comment (danmcp) [reply chris_backend/plugins/services/manager.py]: I assume you mean comma

Comment (NicolasRannou) [reply chris_backend/plugins/services/manager.py]: yes meant comma

File: utils/scripts/pushAllToCUBE.sh [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+function synopsis_show
+{
+        echo "USAGE:"
+        echo "$G_SYNOPSIS"
+}
+
+while getopts E:C:P:p:D: option ; do 
+	case "$option"
+	in
+                D)      PUSHDIR=$OPTARG                 ;;
```
Comment (danmcp) [inline utils/scripts/pushAllToCUBE.sh]: Doesn't align

**Pull request merged**